src/patternfly/components/DataList/examples/data-list-check-action-example.hbs

```diff
@@ -18,6 +21,9 @@
     {{#> data-list-cell}}
       Secondary content
     {{/data-list-cell}}
-    {{#> data-list-action button--attribute='aria-labelledby="check-action-item2 check-action-action2" id="check-action-action2" aria-label="Actions"'}}{{/data-list-action}}
+    {{#> data-list-action button--attribute='aria-labelledby="check-action-item2 check-action-action2" id="check-action-action2" aria-label="Actions"'}}
```

Comment:
did you mean to leave `button--attribute` on here? you might also see if you can add the `{{#> dropdown` to `data-list-action.hbs` except omit the `id` attribute and add it to `{{#> data-list-action` instead.

Reply:
I moved `{{#> dropdown` to `data-list-action.hbs` , and the id is now on data-list-action. But my concern is that I believe the button inside the dropdown needs an ID. It takes in `dropdown-id` and concats it onto `-button` but I'm having trouble with handlebars passing the `data-list-action--ID` into the dropdown id.

If this is not possible I think I may have to add the dropdown back in otherwise it will not have a unique ID. @mcoker

Reply:
You don't have `button--attribute` on any of the other `data-list-action` partials - why do you need it here? And re: the ID, I meant do `{{#> data-list-action id="dropdown-kebab-right-aligned-2"}}` - that should pass `id` in the context of `data-list-action`, which can be consumed by `{{#> dropdown}}` inside of it.

Reply:
I had removed button--attribute not sure why it was showing up. Should be good now